Doc/library/shlex.rst

```diff
@@ -43,15 +43,16 @@ The :mod:`shlex` module defines the following functions:
    string that can safely be used as one token in a shell command line, for
    cases where you cannot use a list.
 
-   This idiom would be unsafe::
+   This idiom would be unsafe:
```

Comment:
Sorry, I forgot to comment on this earlier. Your change looks fine to me. I still need to run the tests locally.